woocommerce/woocommerce-android
# [Woo POS] UI changes on product info dialog

Closes: #12509


### Description

This PR has UI changes as per Joe's suggestion. Specifically, it includes following changes:
1. Adjusted modal dialog width to 75% of the total screen width. This change applies to both the Product Info dialog and the Exit POS dialog.
2. Updated line heights and spacing in the Product Info dialog. To achieve this, I split the existing Primary and Secondary messages into Primary, Secondary, and Tertiary messages.
3. Removed the close icon from the Product Info dialog, as the primary button now handles the close action.

<img width="891" alt="Screenshot 2024-09-04 at 9 53 43 AM" src="https://github.com/user-attachments/assets/168fcf82-3b0b-47b6-ba11-f6db631772e6">



### Testing information


1. Navigate to Product info dialog (More menu -> POS mode -> Click on Simple products banner learn more -> Product info dialog)
2. Ensure the Product info dialog modal is 75% of the total screen width (Also check Exit POS dialog as it uses the same wrapper dialog)
3. Ensure the product info dialog's text spacing and line height are as per design
4. Ensure you don't see close icon in Product info dialog.

### The tests that have been performed

Tested Product info dialog and Exit POS dialog both on Light and Dark mode.

### Images/gif




https://github.com/user-attachments/assets/910e4d57-cc47-451b-830e-f8fd225d1aba





## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
